chainer/training/updater.py

```diff
@@ -260,6 +273,15 @@ def __init__(self, iterator, optimizer, converter=convert.concat_examples,
             if devices['main'] >= 0:
                 optimizer.target.to_gpu(devices['main'])
 
+        # Correct optimizer parameters for new minibatch size
+        optim = optimizer.__class__.__name__
+        if optim in ('Adam', 'AdaGrad', 'RMSprop'):
+            optimizer.eps /= len(devices)
```

Comment:
I found eps is divided by the number of devices here. Should it be multiplication instead? I thought that eps has the same unit as the gradient, and thus it should be scaled linearly with the mini-batch size.